.clang-format

```diff
@@ -0,0 +1,13 @@
+---
+BasedOnStyle: LLVM
+Language: Cpp
+AlignConsecutiveAssignments: true
```

Comment:
This is probably the biggest change. It lines up the equals signs in blocks of assignments. I generally prefer that, but it's easy to drop if others don't.